src/firebase.js

```diff
@@ -0,0 +1,17 @@
+import * as firebase from "firebase"
+import "firebase/firestore"
+
```

Comment:
**DELETE** this file from Github. make sure to delete this also from GitHub history (previous commits).
this type of vars should be in .env file or should be added to .gitignore